src/types/flow.ts

```diff
@@ -0,0 +1,4 @@
+export type FlowStatsFilter = {
```

Comment:
Trying to figure out where this is used outside of the mappers. I'm assuming this is used at implementation on the actual flow page?

Reply:
It's only used in mappers, but in nebula there's an additional `mapFlowStatsFilterTo**EventsFilter**` that uses it

Reply:
Yep, makes sense 👍